pylint_odoo/checkers/modules_odoo.py

```diff
@@ -334,6 +340,14 @@ def visit_import(self, node):
             if isinstance(node.scope(), astroid.Module):
                 self._check_imported_packages(node, name)
 
+    @utils.check_messages('except-pass')
+    def visit_tryexcept(self, node):
+        """Visit block try except"""
+        for handler in node.handlers:
+            if handler.body and isinstance(handler.body[0],
```

Comment:
And `len(handler.body) == 1` because we need validate just that unique sentence.

```python
try:
  code
except:
   pass
   logger  # this case is valid, is other error but is valid for this one
```